snowflake/common_columns.go

```diff
@@ -0,0 +1,57 @@
+package snowflake
+
+import (
+	"context"
+
+	"github.com/turbot/steampipe-plugin-sdk/v3/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v3/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v3/plugin/transform"
+)
+
+func snowflakeColumns(columns []*plugin.Column) []*plugin.Column {
+	return append(columns, commonColumns()...)
+}
+
+// column definitions for the common columns
+func commonColumns() []*plugin.Column {
+	return []*plugin.Column{
+		{
+			Name:        "region",
+			Type:        proto.ColumnType_STRING,
+			Hydrate:     plugin.HydrateFunc(getCommonColumns).WithCache(),
+			Description: "The Snowflake Region in which the account is located.",
+		},
+		{
+			Name:        "account",
```

Comment:
```suggestion
			Name:        "account_id",
```

Should the column name be `account_id`? How does Snowflake refer to their accounts in docs, APIs, etc.?